autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/viewing/AutofillManagementListMode.kt

```diff
@@ -100,7 +102,9 @@ class AutofillManagementListMode : DuckDuckGoFragment(R.layout.fragment_autofill
             viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.STARTED) {
                 viewModel.viewState.collect { state ->
                     binding.enabledToggle.quietlySetIsChecked(state.autofillEnabled, globalAutofillToggleListener)
-                    credentialsListUpdated(state.logins)
+                    state.logins?.let {
```

Comment:
if `state.logins` is null here, we still don't know if we have credentials or not, so ignore it until we do know for sure